peer/CMakeLists.txt

```diff
@@ -49,7 +39,9 @@ target_link_libraries(iroha-main
   http_server_with_cappuccino
   connection_with_grpc
   config_manager
-  pthread
-  keccak
+  ed25519
   hash
+  pthread
```

Comment:
related to this [comment](https://github.com/hyperledger/iroha/pull/178/files#r103535396)